src/docs/transactions.dox

```diff
@@ -148,4 +148,27 @@ WT_SESSION::snapshot with a configuration that includes
 the semantics supported by the drop configuration.
 
 Named snapshots are not durable: they do not survive WT_CONNECTION::close.
+
+@section transaction_timestamps Timestamps
+
+Applications can assign transactions explicit commit timestamps, then read
+as-of a timestamp.  Timestamps are communicated to WiredTiger as hexadecimal
+values with a length configurable at build time (default 8 bytes).
```

Comment:
I see your point, but I don't think applications will care too much about the encoding.  Instead, they will have some native representation of a timestamp with some size (e.g., MongoDB's 8-byte OpTime).  That has to pass through the WiredTiger API somehow, but I don't think we should focus on the size of the encoding -- applications care about how big the actual timestamp can be.